src/pages/sighting/SightingCore.jsx

```diff
@@ -119,18 +132,29 @@ export default function SightingCore({
       /> */}
       <ConfirmDelete
         open={deleteDialogOpen}
-        onClose={() => setDeleteDialogOpen(false)}
+        onClose={() => {
+          setMessageForConfirmDelete(
+            'CONFIRM_DELETE_SIGHTING_DESCRIPTION',
+          );
+          setVulnerableIndividual(null);
+          setDeleteDialogOpen(false);
+        }}
         onDelete={async () => {
           let deleteResults;
           if (pending) {
             deleteResults = await deleteAssetGroupSighting(id);
-          } else {
+          }
+          if (!pending && vulnerableIndividual) {
+            deleteResults = await deleteSighting(id, true);
+          }
+          if (!pending && !vulnerableIndividual) {
             deleteResults = await deleteSighting(id);
           }
           const successful = pending
             ? deleteResults?.status === 204
             : deleteResults;
           if (successful) {
+            setVulnerableIndividual(null);
```

Comment:
I think it makes more sense to reset vulnerable individual in the `if(successful)` block of useDeleteSighting.
https://github.com/WildMeOrg/codex-frontend/blob/79b26e174c17de1512149c9f67ad257e88134bc5/src/models/sighting/useDeleteSighting.js#L32-L37

Reply:
Ok. Done.